modules/win_generic_worker/templates/run-hw-generic-worker-and-reboot.bat.epp

```diff
@@ -53,11 +70,13 @@ shutdown /r /t 0 /f /c "Generic worker has reached idle timeout; rebooting" >> <
 exit
 
 :exit_69
+echo "Generic-worker panic! Issue with enviroment or worker bug. (exit code %GW_EXIT_CODE%)" >> <%= $win_generic_worker::generic_worker_dir %>\generic-worker-wrapper.log
+echo "Removing possibly corrupted files." >> <%= $win_generic_worker::generic_worker_dir %>\generic-worker-wrapper.log
+del /Q /F <%= $win_generic_worker::generic_worker_dir %>\tasks-resolved-count.txt
 del /Q /F <%= $win_generic_worker::generic_worker_dir %>\file-caches.json
 del /Q /F <%= $win_generic_worker::generic_worker_dir %>\directory-caches.json
 del /Q /F <%= $win_generic_worker::generic_worker_dir %>\next-task-user.json
 del /Q /F <%= $win_generic_worker::generic_worker_config %> >> <%= $win_generic_worker::generic_worker_dir %>\generic-worker-wrapper.log
-echo "Generic-worker panic! Issue with enviroment or worker bug. (exit code %GW_EXIT_CODE%)" >> <%= $win_generic_worker::generic_worker_dir %>\generic-worker-wrapper.log
 Goto ErrorReboot

```

Comment:
on posix, we've seen workers reboot "too quickly" and end up thrashing in the 69 state when a taskcluster service was down.
Is that a risk on Windows? On posix, we ended up just stopping the worker (so we have to manually go un-block them since either they're broken or a TC service is down). But I think we'll change it to a 1-2min delay and then reboot so they don't repeat their failure quickly.

Reply:
On Windows we were seeing one of the json files or a single text file  end up with a data that was causing a Go error. I wonder if generic worker was writing to those files when the reboot happened which could possibly cause an issue with the data being written.  Like generic worker exits but the service is still writing out to files when the wrapper script calls a reboot.